modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -1182,6 +1182,26 @@ private static void PullWithFilter(Database database)
             // end::replication-pull-filter[]
         }
 
+        public void TestReplicatorHeartbeatGetSet()
```

Comment:
```suggestion
        public void TestCustomHeartbeat()
```
Change test method name to match iOS and java/android.